README.md

```diff
@@ -47,17 +47,15 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Portfolio] <a name="about-project"></a>
+# 📖 Portfolio <a name="about-project"></a>
```

Comment:
[OPTIONAL] This is optional as it wasn't mentioned by the previous reviewer 🤓  (strongly recommended ), you have a nice README file 📄 but you seem to have removed some required sections of the README, I noticed that you removed the following:
- [Usage](https://github.com/microverseinc/readme-template#usage)
- [Run tests](https://github.com/microverseinc/readme-template#run-tests)
- [Deployment](https://github.com/microverseinc/readme-template#triangular_flag_on_post-deployment)

Please consider adding them to make the README complete and more professional, if you're not sure what to put in these sections you can put `N/A` and/or mention that this section does not apply. But it is required to be present. 😉